Pod/Classes/XExtensionItem.h

```diff
@@ -89,6 +126,13 @@ typedef UIImage *(^XExtensionItemThumbnailProvidingBlock)(CGSize suggestedSize,
 @property (nonatomic) XExtensionItemReferrer *referrer;
 
 /**
+ A block that will be called with the suggested size for the thumbnail image, in points, and the activity type chosen by 
+ the user. It should return an image using the appropriate scale for the screen. Images provided at the suggested size 
+ will result in the best experience.
+ */
+@property (nonatomic, copy) XExtensionItemThumbnailProvidingBlock thumbnailProvider;
```

Comment:
Yep, as @AriX said the goal was to avoid confusion with `NSItemProvider`. Not sure how crucial that is or not.